src/Eluceo/iCal/Component/Event.php

```diff
@@ -270,7 +270,7 @@ public function buildPropertyBag()
                         )
                     )
                 );
-                $propertyBag->set('GEO', $this->locationGeo);
+                $propertyBag->set('GEO', str_replace(',', ';', $this->locationGeo));
```

Comment:
very very naively exchanging the `,` (that is required by the Apple geo property and therefore is required in the geo parameter as well) with a `;`